README.md

```diff
@@ -50,22 +50,32 @@ The following environment variables can be used to configure how the framework e
 
 #### Run Examples on the Web (`wasm32-unknown-unknown`)
 
-Running on the web is still work-in-progress. You may need to enable experimental flags on your browser. Check browser implementation status on [webgpu.io](https://webgpu.io). Notably, `wgpu-rs` is often ahead in catching up with upstream WebGPU API changes. We keep the `gecko` branch pointing to the code that should work on latest Firefox.
+##### WebGL and WebGPU
+
+Running on the web is still work-in-progress. You may need to enable experimental flags on your browser to use WebGPU. Check browser implementation status on [webgpu.io](https://webgpu.io). Notably, `wgpu-rs` is often ahead in catching up with upstream WebGPU API changes. We keep the `gecko` branch pointing to the code that should work on latest Firefox.
```

Comment:
This note is for the WebGPU backend only and not WebGL

Reply:
I meant to keep the "running on the web is still a work-in-progress" part mainly. I can restructure this when I have a chance.